content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,201 @@
+---
+title: "Sourcegraph 4.0"
+publishDate: 2022-09-27T10:00-07:00
+description: Announcing Sourcegraph 4.0
+tags: [blog, release]
+slug: "release/4.0"
+published: true
+heroImage: 
+socialImage: 
+changelogItems:
+---
+
+Sourcegraph 4.0, the latest version of our code intelligence platform, is now available. It includes more than 12 new features and enhancements to help you spend more time in flow and less time manually grokking your codebase. 
+
+Our code intelligence platform semantically indexes and analyzes code and its underlying structure to help you derive knowledge from the codebase. It provides devs and engineering leaders with a complete understanding of the codebase to make faster and better decisions. Our platform enables anyone who creates or interacts with code to save time by helping you understand, fix, and automate changes across your entire codebase.
+
+Read on to discover everything that shipped in Sourcegraph 4.0.  
+
+### 🌅 Code intelligence: more useful, more times daily for every dev
+
+
+<Badge text="Code Search" color="cerise" size="small" onClick={() => window.open("/code-search", "_self")} />
+
+
+#### A faster, simpler search experience 
+We streamlined and simplified the search user interface to make the platform more intuitive. The search homepage now includes contextual tips and recommendations using real data from your instance, including repository names, file names, and authors. This information makes it easier than ever run a successful search query, faster.
+
+Search results are now front and center with the improved interface. The search sidebar is now on the right side of the screen and we removed several non-essential elements to ensure the information you need is easy to access. Filters make it easier and faster for you to answer questions about your code, and the sidebar can be collapsed altogether to reduce noise. 
+
+TODO: Search results are returned faster with our performance improvements using the latest in React, lazy loading, and priority rendering.
```

Comment:
@taylorsperry could you please provide more information here on what the value is - faster load time, faster search query results, etc?